src/invision/store/actions/add-movie.action.js

```diff
@@ -0,0 +1,22 @@
+import { ADD_MOVIE } from "./action.types";
+import { movieMapper } from '../../utils/movie-mapper.util';
+import { movieConverter } from '../../utils/movie-converter.util';
+
+export const addMovieAction = (movie) => {
+  movie = movieConverter(movie);
+  return dispatch => {
+    fetch(`http://localhost:4000/movies?sortBy=release_date&sortOrder=asc`, {
```

Comment:
Do you really need this URL in the **addMovieAction** action?

Reply:
this URL is provided in the API documentation. So I used that.